pyccel/ast/builtins.py

```diff
@@ -355,6 +355,8 @@ class PythonEnumerate(PyccelAstNode):
     __slots__ = ('_element','_start')
     _attribute_nodes = ('_element','_start')
     name = 'enumerate'
+    _class_type = SymbolicType()
```

Comment:
I feel that it should be useful to mention in the CHANGELOG that `_class_type = SymbolicType` for the classes `PythonEnumerate`, `PythonMap`, `PythonRange`, and `PythonType`.

Reply:
Done 02c0511bf